include/unifex/variant_sender.hpp

```diff
@@ -139,7 +144,7 @@ class _sender<Senders...>::type {
   }
 
   friend constexpr auto tag_invoke(tag_t<blocking>, const type&) noexcept {
-    return _variant_sender::max_blocking_kind<cblocking<Senders>...>{}();
+    return _variant_sender::max_blocking_kind<Senders...>{}();
```

Comment:
Should this handle the case where the constexpr blocking_kind is 'maybe' to then go on to individually query the blocking kind of the selected variant type using `visit`?
e.g.
```c++
constexpr auto static_blocking = _variant_sender::max_blocking_kind<Senders...>{}();
if constexpr (static_blocking == blocking_kind::maybe) {
  return visit(blocking, sender.senderVariant_);
} else {
  return static_blocking;
}
```